packages/react-native-client/ios/VideoPreviewView.swift

```diff
@@ -26,12 +27,28 @@ class VideoPreviewView: ExpoView {
                 return
             }
 
-            localVideoTrack =
+            self.localVideoTrack =
                 tracks.first(where: { (key, track) in
                     track is LocalVideoTrack
                 })?.value as? LocalVideoTrack
-            localVideoTrack?.start()
-            videoView?.track = localVideoTrack
+            self.localVideoTrack?.start()
+            self.videoView?.track = self.localVideoTrack
+        }
+    }
+
+    override func willMove(toSuperview newSuperview: UIView?) {
+        super.willMove(toSuperview: newSuperview)
+        if newSuperview == nil {
```

Comment:
Personally I like `guard let _ = newSuperview else { ... }` better because it avoids going deeper in braces but that one is up for discussion.

Reply:
I prefer to use guard when after `else` is one line of code or return statement. In this case it is not applicable.